### DISCUSSION TITLE
[Fedora - Gnome] How can i open files with admin permissions?

### DISCUSSION THREAD

**Discussion by socksprox** [elementary/files#2078] [Q&A] [answered]
And how can i make that when i click on a folder then, it asks me for  pw instead of just not displaying options to delete the file/edit it

**Comment by jeremypw** [comment, marked as answer]
You can open Files with admin permissions from the Pantheon Dock - just secondary click and choose "New Window as Administrator".  An alternative way is to run `io.elementary.files-pkexec` in a terminal.   A third way is to open a new tab with the path `admin://`.

Becoming administrator on demand is the subject of an (old) feature request (#165) but is not simple to implement.

**Reply by socksprox** [reply]
Oh, i dont use elementary os, just the file mgr of it.

**Reply by jeremypw** [reply]
OK, well the second two solutions should still work.

**Reply by Marukesu** [reply]
> Oh, i dont use elementary os, just the file mgr of it.

The first solution should work with any desktop, you can found it by using the secondary click in the Files icon inside the "Applications" menu or in the hidden dock, if files in on it.
